ledger-core/v2/application/cmd/insolar/bootstrap.go

```diff
@@ -25,13 +26,15 @@ func bootstrapCommand() *cobra.Command {
 			gen, err := bootstrap.NewGenerator(configPath, certificatesOutDir)
 			check("bootstrap failed to start", err)
 
-			err = gen.Run(context.Background())
+			err = gen.Run(context.Background(), properNames)
```

Comment:
can just generate "proper names" and don't create an option out of it? If names are not human unreadable then I don't see a reason to keep old variant around.

Reply:
yes, this is only for backward compatibility with launchnet.sh